examples/fastify-typescript-starter/README.md

```
@@ -0,0 +1,87 @@
+
+### Other Commands
+
+```bash
+pnpm typecheck   # Check for TypeScript errors
+pnpm format      # Format code with Prettier
+pnpm build:moose # Build Moose outputs in ./moose/dist
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)